services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/CryptoProvider.java

```diff
@@ -65,6 +66,15 @@
     Security.addProvider(new BouncyCastleProvider());
   }
 
+  /**
+   * Reads the {@link Certificate} and {@link PrivateKey} configured in the application properties.
+   */
+  @PostConstruct
```

Comment:
Hi @elch78, since we are using Constructor Injection, the loading of PrivateKey and Certificate should be already possible in the constructor as the privateKeyStream dependency is injected and available. Any particular reason to do it in PostConstruct?

Reply:
Hi @stevesap . The reason to  not do it in the constructor was just the habit to avoid IO in the constructor because the constructor is normally not mockable. But since this whole class is only an abstracting for the disc IO I also think that the code can be called from the constructor. I'll change that. Then also the members are obsolete and the loaded member can be final.